### PR TITLE
Drawn status item

### PR DESCRIPTION
**Pull request by sdsykes**
Draw the status item in code. Todo: invert colours for dark menu bar setting

### CODE REVIEW

**Comment by sdsykes** [comment]
Oh, it doesn't look exactly like the original - feel free to change the font to something more funky / to your taste

File: WhichSpace/AppDelegate.swift [hunk 1]
```diff
@@ -9,13 +9,14 @@
...
 @NSApplicationMain
-class AppDelegate: NSObject, NSApplicationDelegate {
+class AppDelegate: NSObject, NSApplicationDelegate, NSMenuDelegate {
 
     @IBOutlet weak var window: NSWindow!
     @IBOutlet weak var statusMenu: NSMenu!
 
     var icons = [NSImage]()
-    let statusBarItem = NSStatusBar.systemStatusBar().statusItemWithLength(-1)
+    let statusBarItem = NSStatusBar.systemStatusBar().statusItemWithLength(27)
```
Comment (gechr) [inline WhichSpace/AppDelegate.swift]: I'm very new to Cocoa, so excuse my ignorance, but why 27 rather than `NSVariableStatusItemLength` (i.e. `-1`)?

Comment (sdsykes) [inline WhichSpace/AppDelegate.swift]: That's the width of the status item, which I wanted to specify as I was drawing it myself

Comment (gechr) [inline WhichSpace/AppDelegate.swift]: Ah-hah, of course - if there's no image present, it can't automatically determine the width. Thanks!

**Comment by gechr** [comment]
Thanks a lot for the assistance so far, really appreciate it.

Just got around to testing this and it works very nicely! I'll have a play around with the styling, but looks great so far.

**Pull request merged**